lib/spree/localized_number.rb

```diff
@@ -30,6 +32,8 @@ def localize_number(*attributes)
         end
 
         define_method(:validate_localizable_number) do
+          return unless Spree::Config.enable_localized_number?
+
```

Comment:
Hey @ltrls, thanks for adding this. I am just wondering, it there any reason that we can't return at the top of the `localize_number` method if `Spree::Config.enable_localized_number?` is `false`? I would prefer to not override any attribute setters unless it is necessary...

Reply:
yeah but the `localize_number` method is called when model classes are loaded by Ruby, so I think the Config switch would be effective only when the application is restarted, right? I mean, if I overwrite the setters and in the same runtime I decide to deactive the separator thing, then they wouldn't be un-overwritten before the next restart I think. It's not a big deal as it's not meant to be switched every now and then though...

Reply:
Yeah, I think this is a "set and forget" kind of configuration. I don't think we need to handle toggling it on and off frequently. 

This is related: how do you feel about configuring this via `application.yml` rather than through the admin interface? Pau and I think that maybe the interface is overkill, given that it won't need to be changed often, and it is probably a sysadmin task rather than a instance manager task? I am happy to do the work, but I wanted to check with you first.

Reply:
@ltrls ☝️

Reply:
yes that would be nice. That would avoid the confusion of an administrator (not sysadmin) configuring something that says "you need to restart the Rails application"..!